src/gui/tray/UserLine.qml

```diff
@@ -111,16 +111,27 @@ MenuItem {
                             font.pixelSize: 12
                             font.bold: true
                         }
-                        Label {
-                            id: userStatusMessage
+                        Row {
+                            id: userStatus
+                            spacing: 8
                             visible: model.isConnected && 
-                                     model.serverHasUserStatus && 
-                                     statusMessage !== ""
-                            width: 128
-                            text: statusMessage
-                            elide: Text.ElideRight
-                            color: "black"
-                            font.pixelSize: 10
+                                     model.serverHasUserStatus
+                            Label {
+                                id: emoji
+                                visible: model.statusEmoji !== ""
+                                width: Style.userStatusEmojiSize
+                                text: statusEmoji
+                            }
+                            Label {
+                                id: message
+                                anchors.bottom: emoji.bottom
+                                visible: model.statusMessage !== ""
+                                width: 128
+                                text: statusMessage
+                                elide: Text.ElideRight
+                                color: "black"
+                                font.pixelSize: 10

```

Comment:
Are you sure that is not needed also for the emoji in the previous label ?
I think that we could run into problem if the font inherited from system has a different size than the one you tested with

Reply:
OK, the font pixel is used everywhere at the moment.
I just to want point out that this file is outdated. I am afraid you haven't seen my recent changes - the ones I really would like to get feedback on.